tsdb/chunks/chunks.go

```diff
@@ -67,6 +67,20 @@ type Meta struct {
 	MinTime, MaxTime int64
 }
 
+// Iterator iterates over the chunk of a time series.
+type Iterator interface {
```

Comment:
This isn't used anywhere in this module, is this the right place for it?

Reply:
Hm, yes as it is a nice place that everyone depends on anyway for chunks.

Reply:
This is really the same as `chunkenc.Iterator`